README.md

```diff
@@ -149,12 +149,12 @@ more Emacs-y.
 
    MacOS:
    
-        brew install libxml2 gnutls texinfo
+        brew install gnutls texinfo
         
     To use the installed version of `makeinfo` instead of the built-in 
     (`/usr/bin/makeinfo`) one, you'll need to make sure `/usr/local/opt/texinfo/bin` 
     is before `/usr/bin` in `PATH`.
-    Mojave: `export PKG_CONFIG_PATH="/usr/local/opt/libxml2/lib/pkgconfig"`
+    Mojave: `export PKG_CONFIG_PATH="/Library/Developer/CommandLineTools/Packages/macOS_SDK_headers_for_macOS_10.14.pkg"`
```

Comment:
Does the user need to install this first? Or is only pointing at the `.pkg` sufficient.

Reply:
No I think this should be available once Xcode Command Line tools are installed and they are a requirement for compiling stuff on macOS, you install them with `xcode-select --install`